2022-06-30 08:27:42,219 INFO    MainThread:1779269 [wandb_setup.py:_flush():69] setting env: {}
2022-06-30 08:27:42,219 INFO    MainThread:1779269 [wandb_setup.py:_flush():69] setting login settings: {}
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_setup.py:_flush():69] setting login settings: {}
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_init.py:_log_setup():337] Logging user logs to /home/v-linyukong/work/conform_discriminator/wandb/run-20220630_082742-feykp7et/logs/debug.log
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_init.py:_log_setup():338] Logging internal logs to /home/v-linyukong/work/conform_discriminator/wandb/run-20220630_082742-feykp7et/logs/debug-internal.log
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_init.py:init():370] calling init triggers
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [wandb_init.py:init():419] starting backend
2022-06-30 08:27:42,220 INFO    MainThread:1779269 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-30 08:27:42,228 INFO    MainThread:1779269 [backend.py:ensure_launched():135] starting backend process...
2022-06-30 08:27:42,234 INFO    MainThread:1779269 [backend.py:ensure_launched():139] started backend process with pid: 1779288
2022-06-30 08:27:42,235 INFO    MainThread:1779269 [wandb_init.py:init():424] backend started and connected
2022-06-30 08:27:42,236 INFO    MainThread:1779269 [wandb_init.py:init():472] updated telemetry
2022-06-30 08:27:42,237 INFO    MainThread:1779269 [wandb_init.py:init():491] communicating current version
2022-06-30 08:27:43,006 INFO    MainThread:1779269 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.12.20 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-30 08:27:43,006 INFO    MainThread:1779269 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2022-06-30 08:27:43,098 INFO    MainThread:1779269 [wandb_init.py:init():529] starting run threads in backend
2022-06-30 08:27:43,155 INFO    MainThread:1779269 [wandb_run.py:_console_start():1623] atexit reg
2022-06-30 08:27:43,155 INFO    MainThread:1779269 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2022-06-30 08:27:43,156 INFO    MainThread:1779269 [wandb_run.py:_redirect():1502] Redirecting console.
2022-06-30 08:27:43,162 INFO    MainThread:1779269 [wandb_run.py:_redirect():1558] Redirects installed.
2022-06-30 08:27:43,162 INFO    MainThread:1779269 [wandb_init.py:init():554] run started, returning control to user process
2022-06-30 08:27:43,163 INFO    MainThread:1779269 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2022-06-30 08:27:43,164 INFO    MainThread:1779269 [wandb_run.py:_restore():1565] restore
2022-06-30 08:27:45,668 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 738
  total_bytes: 738
}

2022-06-30 08:27:45,891 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 738
  total_bytes: 738
}

2022-06-30 08:27:46,106 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 738
  total_bytes: 942
}

2022-06-30 08:27:46,207 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 738
  total_bytes: 4678
}

2022-06-30 08:27:46,308 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1214
  total_bytes: 4678
}

2022-06-30 08:27:46,409 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:46,510 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:46,611 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:46,712 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:46,969 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:47,070 INFO    MainThread:1779269 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4678
  total_bytes: 4678
}

2022-06-30 08:27:48,169 INFO    MainThread:1779269 [wandb_run.py:_show_files():1937] logging synced files
